src/njoy21/Version.hpp

```diff
@@ -4,9 +4,10 @@ class Version{
   // Change whenever new capability is implemented
   static constexpr int minorVersion{0};
   // Change whenever merge to master branch is done
-  static constexpr int patchVersion{6};
+  static constexpr int patchVersion{7};
 
-  static std::string version(){
+  static constexpr 
```

Comment:
This change causes a compiler error for me, using gcc@9.2.0 on my Mac.  I can provide the full traceback if you need it.

```
/Users/ngibson/opt/NJOY21/src/njoy21/Version.hpp:10:15: error: invalid return type 'std::string' {aka 'std::__cxx11::basic_string<char>'} of 'constexpr' function 'static constexpr std::string njoy::njoy21::Version::version()'
   10 |   std::string version(){
      |               ^~~~~~~
```

Reply:
Oops, forgot to finish that. Thanks for catching this. It should now be fixed.